controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_webhook_test.go

```diff
@@ -209,11 +214,17 @@ func TestKubeadmControlPlaneValidateUpdate(t *testing.T) {
 	invalidUpdateKubeadmConfigInit := before.DeepCopy()
 	invalidUpdateKubeadmConfigInit.Spec.KubeadmConfigSpec.InitConfiguration = &kubeadmv1beta1.InitConfiguration{}
 
+	validUpdateKubeadmConfigInit := before.DeepCopy()
```

Comment:
If I'm not wrong this variable is introduced but never used in a test

Reply:
It is - the diff just looks strange: https://github.com/kubernetes-sigs/cluster-api/pull/3324/files#diff-0d49cc99aec2899fcc03b149561ff8eeR459

The test that was expecting an error was previously using `validUpdateKubeadmConfigInit`, which I changed.

Reply:
Thanks for the catch - got ahead of my self and only tested one of the stanzas. Should be taken care of now.